### PR TITLE
Fixed issue https://github.com/Azure/azure-xplat-cli/issues/2940

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
The content to be added to Changelog is as follows:
* Added opportunity to add NIC with subnet from another resource group (not the same as VM) using subnet-id option

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MikhailTryakhov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by Krylon360** [comment]
Thanks @MikhailTryakhov This is exactly what I was wanting to see. Was going to submit the PR myself; but been slammed for a bit with other tasks.

**Comment by MikhailTryakhov** [comment]
thanks @Krylon360 
please feel free to ping me with compute or networking tasks :)

**Comment by Krylon360** [comment]
@MikhailTryakhov should I start looking into the Python + PowerShell + Go + .NET SDK's to see if the subnet-id function needs to be addressed as well? I can pull down the Java and Node SDK's afterwards. I'll leave the PHP sdk to someone else. heh.

File: lib/commands/arm/vm/vm._js [hunk 1]
```diff
@@ -75,7 +75,9 @@ exports.init = function (cli) {
     .option('-t, --public-ip-idletimeout <public-ip-idletimeout>', $('the public ip idle timeout specified in minutes' +
                    '\n     This parameter will be used only when creating new public IP'))
     .option('-S, --subnet-id <subnet-id>', $('the subnet identifier' +
-                   '\n     e.g. /subscriptions/<subscriptipn-id>/resourceGroups/<resource-group-name>/providers/Microsoft.Network/virtualNetworks/<vnet-name>/subnets/<subnet-name>'))
+                   '\n     e.g. /subscriptions/<subscriptipn-id>/resourceGroups/<resource-group-name>/providers/Microsoft.Network/virtualNetworks/<vnet-name>/subnets/<subnet-name>' +
```
Comment (amarzavery) [inline lib/commands/arm/vm/vm._js]: Can you add or update a test for this scenario?

**Comment by amarzavery** [comment]
@MikhailTryakhov - The CI is failing. Please take a look over [here](https://travis-ci.org/Azure/azure-xplat-cli/jobs/144980833#L3672).

**Comment by MikhailTryakhov** [comment]
Hi @amarzavery 
all checks have passed - let's finally merge it :)

**Pull request merged**